src/controller/java/src/chip/jsontlv/JsonToTlv.kt

```diff
@@ -43,9 +43,20 @@ import java.util.Base64
  * @param json string representing Json encoded data to be converted into TLV format
  * @throws IllegalArgumentException if the data was invalid
  */
-fun TlvWriter.fromJsonString(json: String): ByteArray {
-  validateIsJsonObjectAndConvert(JsonParser.parseString(json), AnonymousTag)
-  return validateTlv().getEncoded()
+fun TlvWriter.fromJsonStringToByteArray(json: String): ByteArray {
```

Comment:
could you not rename this function name, and just keep as it is, thanks

Reply:
rollback.